staff/lgimeneza/react-calculator/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
WARN! react-caculator was not a challenge, but an on-screen demo. please, remove it from your challenges staff folder.

Reply:
done!